charts/alertmanager/values.yaml

```diff
@@ -81,6 +83,7 @@ persistence:
     - ReadWriteOnce
   size: 50Mi
 
+configFileName: config.yml
```

Comment:
With the other fixes you've applied and evaluation of .Values.config for the if statement we could probably omit this option, WDYT?

Reply:
The reason why we decided to do this described in PR notes. If this is not an important case I agree with you.